universal-login-relayer/lib/integration/ethereum/MessageExecutor.ts

```diff
@@ -2,12 +2,18 @@ import {Wallet, providers} from 'ethers';
 import {SignedMessage} from '@universal-login/commons';
 import IMessageValidator from '../../core/services/validators/IMessageValidator';
 import {messageToTransaction} from '../../core/utils/messages/serialisation';
+import {QueueItem} from '../../core/models/QueueItem';
+import {IExecutor} from '../../core/services/execution/IExecutor';
 
-export class MessageExecutor {
+export class MessageExecutor implements IExecutor<SignedMessage> {
```

Comment:
Let's name it `Executor` or `ExecutionService`

Reply:
Are you sure? My idea was to have `MessageExecutor implements IExecutor<SignedMessage>`, and then introduce `DeploymentExecutor implements IExecutor<Deployment>`

Reply:
Aaaa, ok